lib/srv/reexec.go

```diff
@@ -304,13 +304,6 @@ func RunCommand() (errw io.Writer, code int, err error) {
 	if err != nil {
 		return errorWriter, teleport.RemoteCommandFailure, trace.Wrap(err)
 	}
-	defer func() {
```

Comment:
We don't want to close TTY and PTY before we close the process. This cleanup logic was introduced a few months ago https://github.com/gravitational/teleport/pull/13491
After this function returns, the process ends anyway, so all file descriptors will be closed anyways.